File: 268-missing-number/missing-number.js
/**
 * @param {number[]} nums
 * @return {number}
 */
var missingNumber = function(nums) {
    let n=nums.length;
    let max=(n*(n+1))/2
    let sum=0;

    for(let i of nums){
        sum+=i
    }
    return max-sum
    

};

File: 268-missing-number/missing-number.py
class Solution(object):
    def missingNumber(self, nums):
        """
        :type nums: List[int]
        :rtype: int
        """
        n=len(nums)
        # ans=n
        # for i in range(len(nums)):
        #     ans=ans^i^nums[i]
        # return ans
        sum1=(n*(n+1))//2
        s=0
        for num in nums:
            s+=num
        return (sum1-s)